core/src/Kokkos_Concepts.hpp

```diff
@@ -171,7 +171,9 @@ namespace Kokkos {
         std::is_base_of<detected_t<have_t, T>, T>::value ||    \
         std::is_base_of<detected_t<have_type_t, T>, T>::value; \
     constexpr operator bool() const noexcept { return value; } \
-  };
+  };                                                           \
+  template <typename T>                                        \
+  inline constexpr bool is_##CONCEPT##_v = is_##CONCEPT<T>::value;
```

Comment:
This is a C++17 extension isn't it?

Reply:
Yes, both the addition of inline variables to the core language, and the `_v` versions of the querying traits.

Reply:
We did not change the requirements nor update the CI, so your PR is stalled until then

Reply:
That's fine.  I also labelled it as requires C++17.